### PR TITLE
id3tag.md: add page

### PR DESCRIPTION
**Pull request by Dahie**
add id3tag for editing id3 tags of audio files

### CODE REVIEW

**Comment by agnivade** [comment]
Thanks for your PR ! Please fix the travis syntax errors first, then we'll get to bringing the PR to shape. :)

**Comment by agnivade** [comment]
Also, your page title says `id3lib`, but the command is `id3tag`. If both are the same, lets use a consistent naming to avoid confusion.

**Comment by agnivade** [comment]
@Dahie - It seems you have trailing whitespaces on your lines. I think the CI will pass once you fix those.

File: pages/common/id3tag.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# id3tag
+
+> Tool for reading, writing, and manipulating ID3v1 and ID3v2 tags.
```
Comment (agnivade) [inline pages/common/id3tag.md]: maybe mention "of mp3 files" at the end for clarity ?

File: pages/common/id3tag.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- Set artist and title tag of an MP3:
```
Comment (agnivade) [inline pages/common/id3tag.md]: of an MP3 -> of an MP3 file.

Also, you mention artist and title tag in the description, but the command also sets the album. If you want, we can add another example for album or just modify this description to add album too.

**Comment by Dahie** [comment]
Hi, thanks for the feedback, I split the example into two, first one showing an edit of a single file and the second updates the album for all mp3 within the current folder.

**Pull request merged**